analytics/api/src/main/java/mil/nga/giat/geowave/analytic/partitioner/OrthodromicDistancePartitioner.java

```diff
@@ -70,7 +70,7 @@
 	private String crsName;
 	private transient CoordinateReferenceSystem crs = null;
 	private transient GeometryCalculations calculator;
-	private DimensionExtractor<T> dimensionExtractor;
+	protected DimensionExtractor<T> dimensionExtractor;
```

Comment:
Will make sense in the next commit.  It slipped in this one.  I just noticed it.  Recommend letting it go through.